packages/slidev/node/plugins/monacoTypes.ts

```diff
@@ -43,18 +43,25 @@ export function createMonacoTypesLoader({ userRoot }: ResolvedSlidevOptions): Pl
         ].join('\n')
       }
 
-      const matchLoad = id.match(/^\/\@slidev-monaco-types\/load\/(.*?)&name=(.*)$/)
+      const matchLoad = id.match(/^\/\@slidev-monaco-types\/load(\/.*?)&name=(.*)$/)
       if (matchLoad) {
         const [_, root, name] = matchLoad
         const files = await fg(
           [
-            '**/*.ts',
-            '**/*.mts',
-            '**/*.cts',
+            '**/*.d.ts',
+            '**/*.d.mts',
```

Comment:
Some packages may provide original `.ts` files, like `@slidev/client`. And there are also declaration files like `.d.css.ts`.